index.bs

```diff
@@ -242,7 +246,7 @@ It can also be modified directly.
 <pre class=idl>
 [Exposed=(Window,Worker)]
 interface Sanitizer {
-  constructor(optional SanitizerConfig configuration = {});
+  constructor(optional (SanitizerConfig or SanitizerPresets) configuration = "default");
```

Comment:
I was initially thinking this would be bad because we would want the Sanitizer to be empty, but I actually rather like this. And you can still get an empty sanitizer easily, by using `new Sanitizer({ })`.